rcl/src/rcl/publisher.c

```diff
@@ -203,6 +203,16 @@ rcl_publisher_get_default_options()
   default_options.qos = rmw_qos_profile_default;
   default_options.allocator = rcl_get_default_allocator();
   default_options.rmw_publisher_options = rmw_get_default_publisher_options();
+
+  // Load disable flag to LoanedMessage via environmental variable.
+  bool disable_loaned_message = false;
+  rcl_ret_t ret = rcl_get_disable_loaned_message(&disable_loaned_message);
+  if (ret == RCL_RET_OK) {
+    default_options.disable_loaned_message = disable_loaned_message;
+  } else {
+    default_options.disable_loaned_message = false;
```

Comment:
To call `rcl_reset_error` in `else block` might be better because `rcl_get_disable_loaned_message` sets the error message when returns ERROR, right?

What about
```
  bool disable_loaned_message = false;
  rcl_ret_t ret = rcl_get_disable_loaned_message(&disable_loaned_message);
  if (ret != RCL_RET_OK) {
    // debug log
    rcl_reset_error();
  }
  default_options.disable_loaned_message = disable_loaned_message;
```

Reply:
yeah, right. i will fix this as you suggested.

Reply:
actually i just wanted to use local value only when API succeeds, but i will take some of your suggestion here.